samples/drivers/video/capture_to_lvgl/sample.yaml

```diff
@@ -0,0 +1,10 @@
+sample:
+  name: Video capture to LVGL
+tests:
+  sample.video.capture_to_lvgl:
+    build_only: true
+    tags:
+      - video
+    platform_allow:
```

Comment:
Non blocking: Shouldn't it be `integration_platform` if this is the reference platform you want this sample to be built with ?

Reply:
Thanks. I added 
```
integration_platforms:
      - mini_stm32h743
```